WordPress/Tests/WhiteSpace/OperatorSpacingUnitTest.inc.fixed

```diff
@@ -68,3 +68,14 @@ if ( $a === $b or
 // Instanceof
 if ( MyClass instanceof SomeOtherClass ) {}
 if ( MyClass instanceof SomeOtherClass ) {}
+
+// Insteadof
```

Comment:
Is the whitespace meant to be inconsistent like this?

<img width="596" alt="Screenshot 2019-07-21 at 18 25 49" src="https://user-images.githubusercontent.com/88371/61617218-29d35900-ac62-11e9-9b75-b931a41927a1.png">